scss/components/_media-query.scss

```diff
@@ -41,8 +41,8 @@
 
   // Collpased columns have no gutter.
```

Comment:
Unrelated, but there's a small typo here :)

Reply:
Haha, I'm not optimistic that this will ever be merged anyway